README.md

```diff
@@ -129,16 +129,35 @@ versioned_sidebars/version-0.7.0-sidebars.json
 ```
 
 ### Linking docs
-
-- Remember to include the `.md` extension.
-- Files will be linked to correct corresponding version.
-- Relative paths work as well.
-
-```md
-The [@hello](hello.md#paginate) document is great!
-
-See the [Tutorial](../getting-started/tutorial.md) for more info.
-```
+Relative paths work well. - Files will be linked to correct corresponding version.
+  - PREFER RELATIVE PATHS to be consistent with linking.
+    - **Good Example of linking.**
+      For ex say we are updating a 0.12.0 version doc which is older.
+        ```md
+        A [callback notification](writing_data#commit-notifications) is exposed 
+        ```
+        This automatically resolves to /docs/0.12.0/writing_data#commit-notifications.
+    - **Bad example of linking.**
+      For ex say we are updating a 0.12.0 version doc which is older.
+        ```md
+        A [callback notification](/docs/writing_data#commit-notifications) is exposed 
+        ```
+        This will resolve to the most recent release, specifically /docs/writing_data#commit-notifications . We do not want a 0.12.0 doc page to point to a page from a later release.
+  - DO NOT use next version when linking.
```

Comment:
using caution box syntax to highlight? https://github.com/orgs/community/discussions/16925

Reply:
just saw this. will fix the READ ME in a future PR